### DISCUSSION TITLE
insert edit and delete button in table dataframe

### DISCUSSION THREAD

**Discussion by bobwatcherx** [widgetti/solara#119] [Q&A]
I read your tutorial https://solara.dev/api/dataframe Can you add edit and delete buttons to the table.  and a checkbox to select all data and delete more than one data simultaneously

**Comment by maartenbreddels** [comment]
The table cannot yet container UI elements, but we plan to support that later on.

**Comment by maartenbreddels** [comment]
(I also think this was better suited as an issue)
